types/fundamental-react/lib/Menu/Menu.d.ts

```diff
@@ -14,12 +14,16 @@ export type MenuGroupProps = {
 } & React.HTMLAttributes<HTMLDivElement>;
 
 export type MenuItemProps = {
-    addon?: string;
+    active?: boolean;
+    addonAfter?: string;
     addonBefore?: string;
     addonProps?: any;
     className?: string;
+    disabled?: boolean;
     isLink?: boolean;
-    onclick?: (...args: any[]) => any;
+    onClick?: (...args: any[]) => any;
```

Comment:
Please add the correct type here

Reply:
changed it to `    onClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
`